config/database.js

```diff
@@ -1,6 +1,6 @@
 // config/database.js
 module.exports = {
 
-	'url' : 'MongoDBURIExample' // looks like mongodb://<user>:<pass>@mongo.onmodulus.net:27017/Mikha4ot
+	'url' : 'mongodb://archit:gogators@ds031835.mlab.com:31835/uflacm' // looks like mongodb://<user>:<pass>@mongo.onmodulus.net:27017/Mikha4ot
```

Comment:
make sure you do not commit your DB URI! I now have access to this and can blow up your database. I strongly recommend you change your password now